src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+// A read-only immutable interface to get a printable string representing a bunch of person details.
+
+public interface Printable {
+	public String getPrintableString(); 
+	
+    // Returns a concatenated version of the printable strings of each object.
+    default public String getPrintableString(Printable... printables){
+    	final StringBuilder printableString = new StringBuilder();
+    	for(Printable printable:printables){
+    		printableString.append(printable.getPrintableString());
+    	}
+    	return printableString.toString();
+    }
```

Comment:
I don't think this is a good place to put this method and this method is never called.